ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-aro.yaml

```diff
@@ -65,11 +60,8 @@ tests:
     cluster_profile: azure-qe
     test:
     - as: set-upgrade-releases
```

Comment:
If it is a z-stream upgrade, I do not think we need the `set-upgrade-releases` step.